src/sentry/static/sentry/app/components/events/interfaces/spans/spanBar.tsx

```diff
@@ -323,23 +323,38 @@ class SpanBar extends React.Component<SpanBarProps, SpanBarState> {
       );
     }
 
+    const chevronElement = !isRoot ? (
+      <div
+        style={{
+          marginRight: '2px',
+          width: '5px',
+          textAlign: 'right',
+        }}
+      >
```

Comment:
I'm assuming that inline-styles are used because the implementation will change in the future, otherwise it should be a styled-component at the bottom of the file.

Reply:
Yea. I'm likely to leave a lot of the visual changes with respect to the trace view in @doralchan 's hands.